tests/lint/unittest_lint.py

```diff
@@ -524,6 +524,93 @@ def test_load_plugin_command_line() -> None:
     sys.path.remove(dummy_plugin_path)
 
 
+@pytest.mark.usefixtures("pop_pylintrc")
+@pytest.mark.xfail
```

Comment:
Why does this `xfail` again?

Reply:
That... is a git rebase problem and my human error! This xfail should have been removed in the last commit.